docs/building.md

```diff
@@ -385,6 +385,69 @@ Once you've configured your build directory, you can build with
 LANL Employees - to understand how the project space is built out, see
 https://re-git.lanl.gov/eap-oss/parthenon-project
 
+### LNLL RZAnsel (Homogeneous)
+
+Last verified 04 Jan 2021.
+
+#### Allocate Node
+
+[RZAnsel](https://hpc.llnl.gov/hardware/platforms/rzansel) is a homogeneous cluster consisting of 2,376 nodes with the IBM Power9
+architecture with 44 nodes per core and 4 Nvidia Volta GPUs per node. To
+allocate an interactive node: 
+
+E.g.
+```bash
+$ lalloc 1
+```
+
+#### Set-Up Environment (Optional, but Still Recommended, for Non-CUDA Builds)
```

Comment:
@AndrewGaspar, @jlippuner raises a good point why isn't the build configuration simply optional, as far as I can tell there is nothing in there but links to ninja, cmake, the compiler, and git. Is it because of the dependence of the cuda on the compiler?